cve_bin_tool/checkers/systemd.py

```diff
@@ -38,29 +40,18 @@ class SystemdChecker(Checker):
     """
 
     def get_version(self, lines, filename):
```

Comment:
All the tests passed fine without function overriding 
I'll take a deeper look at if we need to keep it after this PR is merged.